src/wp-content/themes/twentyfourteen/inc/featured-content.php

```diff
@@ -268,7 +268,7 @@ public static function delete_post_tag( $tag_id ) {
 
 		$settings['tag-id'] = 0;
 		$settings           = self::validate_settings( $settings );
-		update_option( 'featured-content', $settings );
+		update_option( 'featured-content', $settings, 'on' );
```

Comment:
Why `on` added here?

Reply:
It is the default setting so I set it to be clear as to what is value

Reply:
If this is the case then we should update other instances in core. WDYT?

Reply:
Yes I am looking for them
I feel it is good practice to set this so it clear

Reply:
I have found load more 
I have not set all to 'on' but rather used my judgment and turned off some (ie just admin settings)
So this will need to be reviewed

Reply:
Let's hold off on making any changes to default themes, since those need to support older versions of WordPress. Apologies for missing this in my last review.

Reply:
Agree with @joemcgill because of old version support we should not update bundle theme.

Reply:
I have removed and moved this change to #61103